lib/trino-plugin-toolkit/src/test/java/io/trino/plugin/base/security/TestFileBasedSystemAccessControl.java

```diff
@@ -500,6 +503,15 @@ public void testTableRulesForCheckCanDropTable()
         assertAccessDenied(() -> accessControl.checkCanDropTable(BOB, new CatalogSchemaTableName("some-catalog", "bobschema", "bobtable")), DROP_TABLE_ACCESS_DENIED_MESSAGE);
     }
 
+    @Test
```

Comment:
this tests are much simpler than `TestFileBasedSystemAccessControl.java`. Why so?

Reply:
based on above `testTableRulesForCheckCanDropTable`